src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -0,0 +1,295 @@
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <ScrollViewer  
+            VerticalScrollBarVisibility="Auto" 
+            Background="#535353" 
+            MaxHeight="290">
+            <ItemsControl Name="SearchResultsListBox"
+                              ItemsSource="{Binding Path=LocalPackages}"
+                              HorizontalContentAlignment="Stretch"
+                              VirtualizingPanel.ScrollUnit="Pixel"
+                              BorderThickness="0"
+                              Padding="0">
+
+                <ItemsControl.ItemTemplate>
+
+                    <DataTemplate>
+                        <!--This border shouldn't be wider then installed packages window.
+                        That's why bind its' width to ItemsPresenter's width. 
+                        ItemsPresenter is UI container for ListBox items.
+                        ItemsPresenter can't be wider then visible part of the window.-->
+                        <Border BorderBrush="#535353"
+                                BorderThickness="0,0,0,1"
+                                Background="#535353"
+                                Width="{Binding Path=ActualWidth, RelativeSource={RelativeSource FindAncestor, 
+                                                                            AncestorType={x:Type ItemsPresenter}}}">
+                            <Expander Name="PackageHeader"
+                                              IsExpanded="{Binding Path=Model.TypesVisibleInManager}"
+                                              Style="{StaticResource InstalledPackagesExpanderStyle}">
+                                <Expander.Header>
+                                    <Grid x:Name="PackageGrid"  >
+                                        <DockPanel HorizontalAlignment="Stretch"
+                                                       >
+                                            <StackPanel HorizontalAlignment="Left"
+                                                            Orientation="Horizontal"
+                                                            Margin="10,5,5,7">
+                                                <TextBlock Text="{Binding Path=Model.Name}"
+                                                               ToolTip="{Binding Path=Model.Name}"
+                                                               FontSize="11"
+                                                               Width="150"
+                                                               TextTrimming="CharacterEllipsis"
+                                                               Margin="15,0,10,0"
+                                                               Foreground="#E5E5E5"></TextBlock>
```

Comment:
https://github.com/DynamoDS/Dynamo/blob/bdd63b69c365d5ce5b07ff4ef20649983d49e074/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml#L2273

this one too...